fix: CameraDisplayViewController 사진 비율 및, Aligment Center Cell  코드 추가

Do-hyun-Kim opened the pull request:
## 작업 내용 🧑‍💻

- DisplayEditCollectionViewCell Aligmnet Center로 정렬 하도록 UIEdgeInsets inset 추가(left, right)
- 8글자 초과시 팝업창 띄우는 코드 distinctUntilChanged() 로 인해 중복 글자 입력시에 이벤트 무시되어 코드 제거 or 시간 간격 수정
- DisplayView Image Content Mode 원본 비율 유지 하도록 Content Mode 수정
- 추가 작업으로  WebViewController에서 BibbiNavigationBar 추가하였습니다. -> WebViewNavigationBar가 제거되었기에

## 변경 로직 ⚒️

- DisplayView ContentMode를 추가 하였습니다.
- DisplayEditCollectionViewCell 에서 Center 정렬 하도록 left, right inset 주도록 하였습니다.
- distinctUntilChanged() 로인해 중복 글자 입력시 이벤트가 무시되어 팝업창이 안띄워 지었기에 제거 하였습니다 or 시간 간격이 1초가 조금 긴것 같아 수정하였습니다.

Do-hyun-Kim committed:
fix: CameraDisplayViewController DisplayEdit Cell 가운데 정렬 하도록 Cell Inset 추가
- 텍스트 초과시 팝업창 띄우는 Action distinctUntilChagned 이벤트 무시하는 부분 코드 제거 or 시간 변경
- 사진 비율 조정 코드 추가

Do-hyun-Kim committed:
feat: master.key 임시 추가

YuGeonHui commented on Tuist/master.key:
```diff
@@ -0,0 +1 @@
+jW4XBNfSvzeHt3DjHI4ddss+0mppy7vB6wh5OLnneDY=
```
이 파일 들어가도 괜찮나요 ??

Do-hyun-Kim replied:
github Action 테스트로 잠시 넣어둔거라 camera issue 수정후에 제거해놓겠습니다 :)

akrudal commented on .gitignore:
```diff
@@ -6,7 +6,6 @@
 xcuserdata/
 
 .env
-master.key
```
이 마스터키 파일 왜 깃이그노어에서 제거한건가요?

Do-hyun-Kim replied:
github Action 테스트 하기 위해 잠깐 Repository에 추가하였습니다.

akrudal commented on 14th-team5-iOS/App/Sources/Presentation/Camera/CameraDisplayViewController.swift:
```diff
@@ -231,9 +232,9 @@ public final class CameraDisplayViewController: BaseViewController<CameraDisplay
         displayEditTextField.rx
             .text.orEmpty
-            .distinctUntilChanged()
+            .debug("8글자 이내로 입력이 가능해여 Check")
```
debug는 지워도 될것같아요!

Do-hyun-Kim replied:
네넵 지우겠습니다 :)

akrudal commented on 14th-team5-iOS/App/Sources/Presentation/Camera/CameraDisplayViewController.swift:
```diff
@@ -231,9 +232,9 @@ public final class CameraDisplayViewController: BaseViewController<CameraDisplay
+            .debug("8글자 이내로 입력이 가능해여 Check")
             .map { ($0.count > 8) }
-            .debounce(.seconds(1), scheduler: MainScheduler.instance)
+            .throttle(.milliseconds(300), scheduler: MainScheduler.instance)
```
textField는 입력에 대한 중복 방지라면 debounce가 맞지 않나요?! throttle로 바꾼 이유가 있나요?

Do-hyun-Kim replied:
경미님 말씀처럼 `debounce`로 수정하겠습니다 감사합니다 :D !!

akrudal reviewed (dismissed):
고생하셨습니당! 마스터키 파일 올려도 되는지 확인 한 번만 해주세용

Do-hyun-Kim committed:
feat: ProfileView 기본 이미지 적용 및 NickName 변경시 기본 이미지 일 경우 Profile 이미지 Redraw 하도록 구현
- WebContentViewController NavigationBar PopViewController 코드 추가
- Notification+Ext Name 추가
- 코드 리뷰 적용

Do-hyun-Kim committed:
fix: ProfileViewController DefaultImage 여부 값 확인 State 추가
- CameraViewController Account State 분기 처리

YuGeonHui reviewed (dismissed):
고생하셨습니다

Do-hyun-Kim committed:
- ProfileViewReactor updateNickNameProfile Action 추가
- ProfileViewController isDefaultProfile, Notifcation Userinfo, 통해 NickNameImage 바인딩 및 Default Image Check
- Notification+Ext DidFinshProfileImageUpdate 추가
- CameraViewController Profile Update UserInfo 전달 코드 추가
- AccountNickNameViewController 닉네임 첫글자 및 Profile Update UserInfo 전달 코드 추가

Do-hyun-Kim committed:
feat: 초기 진입시 DefaultProfile 여부 확인 체크 로직 추가

Do-hyun-Kim committed:
Merge branch 'develop' of https://github.com/depromeet/14th-team5-iOS into fix/#273-camera-display-bug-modify

* 'develop' of https://github.com/depromeet/14th-team5-iOS: (34 commits)
  Update README.md
  fix: 위젯 url 서버 주소 분기
  fix: 위젯 서버로 부터 오는 이미지 회전 수정
  fix: screts APP_NAME, WIDGET_NAME 추가
  refactor: 친구 초대 화면에서 링크 공유 뷰 분리 구현 (#267)
  feat: DEV 프로비저닝 프로파일 교체 (#267)
  fix: App_Name, WidetExtension 변경
  fix: 캘린더 피드 셀에서 알 수 없는 가족의 닉네임과 이미지가 제대로 출력되지 않는 문제 수정 (#267)
  fix: fix the navigation(#286)
  fix: remove refresh at familycollectionview(#284)
  fix: fix the timer(#284)
  delete: 화면 진입 시 memberInfo 호출 구문 삭제 (#279)
  feat: 가족 생성 시 Repo에 FamilyID 저장 로직 추가 (#279)
  feat:  가족생성 및 초대화면 추가에 따른 기존 로직 수정 및 추가 개발
  feat: 코드 리뷰 반영 (#279)
  feat: 초대 링크 호출에 IndicatorView 추가 (#279)
  feat: 친구 초대 화면 진입 시 getMemberInfo 메서드 호출 구문 추가 (#279)
  fix: 생일 뱃지가 스트로크에 가려지는 문제 수정(#279)
  feat: 홈 화면 진입 시 getMemberInfo 메서드 호출 구문 추가 (#279)
  fix: MemberInfo 파싱 에러 수정 (#279)
  ...

# Conflicts:
#	14th-team5-iOS/Core/Sources/Extensions/UserDefaults+Ext.swift
